statsmodels/multivariate/plots.py

```diff
@@ -0,0 +1,141 @@
+import matplotlib.pyplot as plt
```

Comment:
matplotlib import needs to be `try ... except` protected because it is not a required dependency

Reply:
still todo, causes import error on Travis with python 2 without matplotlib